internal/frontend/cuefrontend/args/resolvable.go

```diff
@@ -38,7 +38,7 @@ var validKeys = []string{
 	"fromServiceEndpoint",
 	"fromServiceIngress",
 	"fromResourceField",
-	"experimentalFromSecret",
+	"fromKubernetesSecret",
```

Comment:
It's good that internally we switch to fromKubernetesSecret, but lets keep `experimentalFromSecret` around in the frontend for a bit, as there are uses of it that would be broken by this change. You can add a 2nd case here and have both parse into ev.fromKubernetesSecret.

Reply:
Sure, I updated the PR